src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -131,7 +131,7 @@ public Task<string> StartNewAsync(string orchestratorFunctionName, object input)
         /// <param name="instanceId">The ID of the orchestration instance to terminate.</param>
         /// <param name="reason">The reason for terminating the orchestration instance.</param>
         /// <returns>A task that completes when the terminate message is enqueued.</returns>
-        public async Task TerminateAsync(string instanceId, string reason)
+        public virtual async Task TerminateAsync(string instanceId, string reason)
```

Comment:
Let's keep the implementation of `TerminateAsync` here but create a corresponding abstract method in the abstract base class.